stdlib/public/core/StringBridge.swift

```diff
@@ -413,10 +413,40 @@ private func _getCocoaStringPointer(
   return .none
 }
 
+#if arch(arm64)
+//11000000..payload..111
+private let constantTagMask:UInt =
+  0b1111_1111_1000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0111
+private let expectedConstantTagValue:UInt =
+  0b1100_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0111
+#endif
+
+@inline(__always)
+private func formConstantTaggedCocoaString(
+  untaggedCocoa: _CocoaString
+) -> AnyObject? {
+#if !arch(arm64)
+  return nil
+#else
+
+  let constantPtr = unsafeBitCast(untaggedCocoa, to: UnsafeRawPointer.self)
```

Comment:
Err, I don't think this is how we generally do ptr_to_int. Elsewhere we do `Builtin.reinterpretCast`, but maybe there is a more correct/safer way.

cc @atrick and @eeckstein